### PR TITLE
Added Rich Text Editor

### PR DESCRIPTION
**Pull request by Manpreet-hub**
- Added Notes Card Screen
- Added Editor Constants
- Added Rich Text Editor Component

P.S Please ignore commented code, it will be used in the next PR.

### CODE REVIEW

**Review by ajay-mandaviya — APPROVED** [review]
Everything looks good ,
good to merge

File: src/components/TextEditor/TextEditor.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import { useNotes } from "../../contexts/note-context";
+import { modules, formats } from "../../constants/editor-constant";
+const TextEditor = () => {
+  const [note, setNote] = useState({ title: "", content: "" });
+  const { noteState, noteDispatch } = useNotes();
+
+  // const inputHandler = (e) => {
+  //   setNote(e);
+  //   console.log(note);
+  // };
```
Comment (prathmeshjagtap) [inline src/components/TextEditor/TextEditor.jsx:14]: you can remove commented code

File: src/components/TextEditor/TextEditor.jsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+          noteDispatch({ type: "ADD_NOTE", payload: note });
+        }}
+      >
+        <input
+          className="notes-title"
+          placeholder="Title"
+          name="title"
+          value={note.title}
+          onChange={(e) =>
+            // noteDispatch({ type: "SET_TITLE", payload: e.target.value })
```
Comment (prathmeshjagtap) [inline src/components/TextEditor/TextEditor.jsx:34]: remove commented code

File: src/components/TextEditor/TextEditor.jsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+            setNote({ ...note, title: e.target.value })
+          }
+        />
+        <ReactQuill
+          theme="snow"
+          value={note}
+          // onChange={inputHandler}
+          // onChange={(e) => setNote({ ...note, content: e.target.value })}
```
Comment (prathmeshjagtap) [inline src/components/TextEditor/TextEditor.jsx:42]: same here

**Review by prathmeshjagtap — APPROVED** [review]
Rest code is good and well structure

File: src/components/TextEditor/TextEditor.jsx (hunk 2 above)
Comment (Manpreet-hub) [reply src/components/TextEditor/TextEditor.jsx:34]: yes, but it will be used in the next PR.

**Pull request merged**